### ISSUE TITLE
Exporting raw merged cpu pprof file，used for pgo

### ISSUE THREAD

**Issue by yongjichen** [grafana/pyroscope#2233]
#### Is your feature request related to a problem? Please describe.

I want to build golang program with pgo,  so download pprof file from exporting data. When building, it report error: **failed to find CPU samples count or CPU nanoseconds value-types in profile.**  It seams like different from raw pprof file of golang CPUProfile.

#### Describe the solution you'd like

Can download raw merged cpu pprof file for go building with pgo.

**Comment by ly0511** [comment]
I have the same problem，did you find a solution？

**Comment by yongjichen** [comment]
> I have the same problem，did you find a solution？

no, I use runtime profile instead of pyroscope.  I hope pyroscope will fix it.

**Comment by kolesnikovae** [comment]
I'm afraid but the exported pprof does not contain the information necessary for PGO. I guess it should be fixed as part of https://github.com/grafana/grafana/issues/71705

**Comment by kolesnikovae** [comment]
[How to use PGO and Grafana Pyroscope to optimize Go applications](https://grafana.com/blog/2024/03/11/how-to-use-pgo-and-grafana-pyroscope-to-optimize-go-applications)
[Export a profile for Go PGO](https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/profile-cli/#export-a-profile-for-go-pgo)

I'm cleaning up issues. @yongjichen please feel free to reopen the issue if you feel it's not addressed
